src/footer/js/footer-breadcrumbs.js

```diff
@@ -27,7 +28,7 @@ export default (data) => {
     });
 
     return $('div', {class: `${prefix}`},
-      $('a', {href: 'https://www.esri.com/en-us/home', class: `${prefix}--pin`}),
+      $('a', {href: 'https://www.esri.com/en-us/home', class: `${prefix}--pin`}, 'Esri Home'),
       $('ul', {class: `${prefix}--list`}, $breadCrumbs)
```

Comment:
this string should read in a variable as opposed to hard coding "Esri Home", otherwise you wont be able to translate it